### PR TITLE
Normalize kernel read type

### PR DESCRIPTION
**Pull request by NDStrahilevitz**
## Initial Checklist

- [x] There is an issue describing the need for this PR.
- [x] Git log contains summary of the change.
- [x] Git log contains motivation and context of the change.
- [ ] If part of an EPIC, PR git log contains EPIC number.
- [ ] If part of an EPIC, PR was added to EPIC description.

## Description (git log)

Copy and paste the git log here.

Related #2177

## Type of change

- [ ] Bug fix (non-breaking change fixing an issue, preferable).
- [ ] Quick fix (minor non-breaking change requiring no issue, use with care)
- [x] Code refactor (code improvement and/or code removal)
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

## How Has This Been Tested?

1. Run `tracee-ebpf -t e=security_kernel_read_file -o json`
2. Install a kernel module
3. Confirm that the `type` argument is 2
4. Run the same with table printer, confirm the `type` argument is `kernel-module`

## Final Checklist:

Pick "Bug Fix" or "Feature", delete the other and mark appropriate checks.

- [x] I have made corresponding changes to the documentation.
- [x] My code follows the style guidelines (C and Go) of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented all functions/methods created explaining what they do.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix, or feature, is effective.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published before (#2565).

## Git Log Checklist:

My commits logs have:

- [x] Subject starts with "subsystem|file: description".
- [x] Do not end the subject line with a period.
- [x] Limit the subject line to 50 characters.
- [x] Separate subject from body with a blank line.
- [x] Use the imperative mood in the subject line.
- [x] Wrap the body at 72 characters.
- [x] Use the body to explain what and why instead of how.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -104,3 +104,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => github.com/NDStrahilevitz/tracee/types v0.0.0-20230111143732-47176d6b219f
```
Comment (NDStrahilevitz) [inline go.mod:108]: Will remove once #2565 is merged

**Comment by AsafEitani** [comment]
LGTM - does that fixes the backporting problem we had with this event?
In some versions of Ubuntu they backported the enum from a newer version.

**Review by AsafEitani — APPROVED** [review]

**Comment by NDStrahilevitz** [comment]
> LGTM - does that fixes the backporting problem we had with this event? In some versions of Ubuntu they backported the enum from a newer version.

I guess not since I kept the mapping between number to enum the same as it was before, just using the new enum I defined in `trace.go` instead of a string directly.
I guess we can check the OS version of the host with `uname -r` and handle that case though, but that should be in another PR.

**Comment by rafaeldtinoco** [comment]
> can check the OS version of the host with `uname -r`

the OSInfo package has this information already, fyio.

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

**Pull request merged**